src/reynir_correct/errtokenizer.py

```diff
@@ -2402,6 +2425,32 @@ def number_error(
             at_sentence_start = False
 
 
+def late_fix_merges(
+    token_stream: Iterable[CorrectToken],
+    db: GreynirBin,
```

Comment:
The parameters db, token_ctor and only_ci can be deleted

Reply:
Done